lib/config/presets/parse.ts

```diff
@@ -45,7 +45,8 @@ export function parsePreset(input: string): ParsedPreset {
   if (str.includes('(')) {
     params = str
       .slice(str.indexOf('(') + 1, -1)
-      .split(',')
+      .split(/,\s*(?=(?:(?:[^"]*"){2})*[^"]*$)/)
```

Comment:
this can cause harm because of backtracking, so a no-go.

Reply:
@zharinov Idea? Would like to avoid lookahead /loockbehind because of possible backtracking. This can cause harm because it's user input.